src/opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -61,16 +61,21 @@ namespace Opm {
           m_inputNnc(          m_inputGrid, deck),
           m_gridDims(          deck ),
           field_props(         deck, m_runspec.phases(), m_inputGrid, m_tables),
+          aquifer_config(this->m_tables, this->m_inputGrid, deck, this->field_props),
           m_simulationConfig(  m_eclipseConfig.getInitConfig().restartRequested(), deck, field_props),
           m_transMult(         GridDims(deck), deck, field_props),
           tracer_config(       m_deckUnitSystem, deck)
     {
+        if (this->aquifer().hasNumericalAquifer()) {
```

Comment:
Here is where the comment https://github.com/OPM/opm-common/pull/2252#discussion_r567847674 is referring to. @joakim-hove .